be/models.py

```diff
@@ -1,18 +1,11 @@
 from pydantic import BaseModel, Field
-from typing import Literal
+from typing import Generic, List, Literal, TypeVar
 
+T = TypeVar("T")  # Datasource data type
+A = TypeVar("A")  # Datasource aggregation type
```

Comment:
Does it make sense to restrict these generic types?
```
T = TypeVar("T", MaveDBData)
A = TypeVar("A", MaveDBAggregationResponse)
```

Reply:
I thought about doing this; however, TypeVar requires at least two classes in a list to work (it throws an error with just one class).

I'll take a note of this and I'll do this in the next PR when I submit the DepMap classes.

Reply:
sounds good, thanks